### PR TITLE
Instead of spdlog's bundled fmt re-use it from CPM.cmake

### PR DESCRIPTION
**Pull request by jhasse**
I think this is a good example. If someone wants to stick to the bundled one it's easier to remove the OPTION than to add it otherwise.

### CODE REVIEW

**Review by TheLartians — COMMENTED** [review]
Hey, thanks for the PR! I see how this can be helpful for projects using both libraries.
However, for the sake of a minimal example I think I prefer keeping the one-liner here. How about additionally adding the explicit `fmt` example below in a comment with a short explanation instead?

**Comment by jhasse** [comment]
like this?

File: examples/spdlog/CMakeLists.txt [hunk 1]
```diff
@@ -8,6 +8,16 @@ include(../../cmake/CPM.cmake)
 
 CPMAddPackage("gh:gabime/spdlog@1.8.2")
 
+# spdlog uses fmt and bundles that dependency. If you want to use fmt anyway,
+# you can let spdlog re-use fmt from CPM.cmake like this:
```
Comment (TheLartians) [inline examples/spdlog/CMakeLists.txt:12]: I feel this rephrase makes it clearer to the reader what's happening.

```suggestion
# spdlog uses fmt and bundles that dependency. If you want to use fmt in your 
# project as well, you can let spdlog re-use fmt from CPM.cmake like this:
```

**Review by TheLartians — COMMENTED** [review]
Exactly! Though maybe we can make the comment a little more descriptive as suggested.

**Review by TheLartians — APPROVED** [review]
Thanks for the changes!

**Review by TheLartians — DISMISSED** [review]

**Review by TheLartians — APPROVED** [review]
CMake-format screwed up the code example, I had to disable formatting for that part.

**Pull request merged**